central/auth/service/convert.go

```diff
@@ -0,0 +1,88 @@
+package service
```

Comment:
take a look at this https://github.com/stackrox/stackrox/pull/8195 from @md2119.  he started moving the converters to a single location.  Might be worth considering.